spec/lib/tasks/find_possible_duplicate_accounts_spec.rb

```diff
@@ -0,0 +1,159 @@
+require 'rails_helper'
+require 'rake'
+
+RSpec.describe "duplicate_accounts:find", type: :rake do
```

Comment:
See https://github.com/openstax/tutor-server/pull/1229/files for a spec helper I wrote for rake files -- you could pull it in here and reduce some of the code (and probably the top two `before` calls)